mqtt/mqtt-broker/Cargo.toml

```diff
@@ -18,7 +18,6 @@ humantime-serde = "1.0"
 lazy_static = "1.4"
 openssl = "0.10"
 pin-project = "0.4"
-proptest = { version = "0.9", optional = true }
```

Comment:
I guess without this line we cannot run model based tests. could you check please?